src/MuTalk-Tests/MTMethodMutationTest.class.st

```diff
@@ -19,12 +19,14 @@ MTMethodMutationTest >> testAccessing [
 	operator := MTReplacePlusWithMinusMutantOperator new.
 	modifiedSource := operator
 		                  modifiedSourceFor: compiledMethod
-		                  number: 1.
+		                  number: 1
+		                  newExpression: operator newExpression.
```

Comment:
Maybe we could have a method

```smalltalk
MTOperator >> modifiedSourceFor: aMethod number: aNumber
    self modifiedSourceFor: aMethod number: aNumber newExpression: self newExpression
```

to avoid duplication right?